aws-opentelemetry-distro/src/amazon/opentelemetry/distro/aws_opentelemetry_distro.py

```diff
@@ -19,13 +36,15 @@ def _configure(self, **kwargs):
             OTEL_EXPORTER_OTLP_METRICS_DEFAULT_HISTOGRAM_AGGREGATION environment variable. Need to work with upstream to
             make it to be configurable.
         """
+        os.environ.setdefault(OTEL_EXPORTER_OTLP_PROTOCOL, "http/protobuf")
+
         super(AwsOpenTelemetryDistro, self)._configure()
+
+        os.environ.setdefault(OTEL_PROPAGATORS, "xray,tracecontext,b3,b3multi")
+        os.environ.setdefault(OTEL_PYTHON_ID_GENERATOR, "xray")
         os.environ.setdefault(
```

Comment:
q: do we need to explicitly specify `OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4316` as default endpoint here as well? Will the global SpanExporter also be initiated with the default HTTP endpoint?

Reply:
No, default endpoint is tied to choice of protocol: https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/#otel_exporter_otlp_endpoint & https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-python+4318&type=code

Testing done will show this as well (working on it)